### PR TITLE
GitHub Issue NOAA-EMC/GSI#85 bug fix 

### PR DESCRIPTION
**Pull request by ShunLiu-NOAA**
GitHub Issue NOAA-EMC/GSI#85 bug fix for checking ges_z allocation status in setupt.f90

### CODE REVIEW

**Comment by MichaelLueken-NOAA** [comment]
Hi Shun,

Who would you like me to assign for review?

Thanks,

Mike

**Comment by ShunLiu-NOAA** [comment]
Please let Cory review the ticket.

Shun

**Review by CoryMartin-NOAA — APPROVED** [review]

File: src/gsi/setupt.f90 [hunk 1]
```diff
@@ -1375,7 +1375,7 @@ subroutine init_vars_
         varname='th2m'
         call gsi_bundlegetpointer(gsi_metguess_bundle(1),trim(varname),rank2,istatus)
         if (istatus==0) then
-            if(allocated(ges_z))then
+            if(allocated(ges_th2m))then
```
Comment (MichaelLueken-NOAA) [inline src/gsi/setupt.f90:1378]: On line 1378, please replace ges_th2m with ges_th2.  There is no variable, ges_th2m, so the code fails to compile.

File: src/gsi/setupt.f90 [hunk 2]
```diff
@@ -1393,7 +1393,7 @@ subroutine init_vars_
         varname='q2m'
         call gsi_bundlegetpointer(gsi_metguess_bundle(1),trim(varname),rank2,istatus)
         if (istatus==0) then
-            if(allocated(ges_z))then
+            if(allocated(ges_q2m))then
```
Comment (MichaelLueken-NOAA) [inline src/gsi/setupt.f90:1396]: On line 1396, please replace ges_q2m with ges_q2.  There is no variable, ges_q2m, so the code fails to compile.

File: src/gsi/setupt.f90 [hunk 3]
```diff
@@ -1284,7 +1284,7 @@ subroutine init_vars_
      varname='ps'
      call gsi_bundlegetpointer(gsi_metguess_bundle(1),trim(varname),rank2,istatus)
      if (istatus==0) then
-         if(allocated(ges_z))then
```
Comment (MichaelLueken-NOAA) [inline src/gsi/setupt.f90:1287]: With the removal of ges_z on lines 1287, 1378, and 1396, this variable is no longer used and is being flagged while compiling in debug mode.  Please remove ges_z on line 325.

**Review by MichaelLueken-NOAA — COMMENTED** [review]
Hi Shun,

I have completed my review of your changes.  The changes cause the code not to compile.  The reason being that there are no variables, ges_th2m or ges_q2m.  Please replace these with ges_th2 and ges_q2, respectively.  The regression tests were run and all passed successfully.  While  compiling in debug mode, ges_z on line 325 of setupt.f90 was identified as unused, following these changes.  Please remove this unused variable.

To make these changes:

1) git clone --recursive git@github.com:ShunLiu-NOAA/GSI.git update
2) cd update
3) vi src/gsi/setupt.f90
4) Make the requested changes.
5) git add src/gsi/setupt.f90
6) git commit --amend
7) git push origin master --force

If you have any questions or experience any issues, please let me know.

Thanks,

Mike

**Comment by ShunLiu-NOAA** [comment]
Mike,

I updated my master. Thank you.

Shun

**Comment by MichaelLueken-NOAA** [comment]
The due date for feedback from the review committee has past.  This work will now be merged to NOAA-EMC/GSI master.

**Pull request merged**